### PR TITLE
First version of the mapping for MID

### PR DESCRIPTION
**Pull request by dstocco**
This is the initial porting of the mapping for MID.
It actually constitutes the first pull-request for MID.

### CODE REVIEW

File: Detectors/MUON/CMakeLists.txt [hunk 1]
```diff
@@ -1 +1,18 @@
+# **************************************************************************
```
Comment (sawenzel) [inline Detectors/MUON/CMakeLists.txt]: @dberzano : What do we do with such copied copyright notices. Should we adjust them to the new one?

**Comment by sawenzel** [comment]
Thanks for this very clean PR. I would just like to understand why everything is green while the log indicates 3 failing tests? Did you force push recently that triggered a successful rebuild (and the log is actually out of date)? (@dberzano)

File: Detectors/MUON/CMakeLists.txt (hunk 1 above)
Comment (dberzano) [reply Detectors/MUON/CMakeLists.txt]: Definitely. If you agree, I am changing the check to include `CMakeLists.txt` as well (we only check .cxx, .h & at the moment). OK?

**Comment by dstocco** [comment]
Hi Sandro,
I was forced to do a force push on my branch on Tuesday (since a pending PR was accepted in the meanwhile and this conflicted with mine). But everything was green since then.
The log by alibuild popped up 10 hours ago...but I'm not modifying the commit since Tuesday. So I have no idea why alibuild suddenly woke up....

Cheers,
Diego

**Comment by dberzano** [comment]
@sawenzel @dstocco sometimes builds fail temporarily. Status temporarily turns red, and log is showed for future reference. Green states at the end are all that matters. The fact that the log is broken is an acknowledged issue we've got to fix.

File: Detectors/MUON/CMakeLists.txt (hunk 1 above)
Comment (dstocco) [reply Detectors/MUON/CMakeLists.txt]: Dear @sawenzel and @dberzano ,
sorry I missed the point here. Should I modify the copyright notice using e.g. the one that I find in Framework/Core/CMakeLists.txt?

In addition, I did not add a copyright in:
Detectors/MUON/MID/CMakeLists.txt
Detectors/MUON/MID/Base/CMakeLists.txt
I'm actually not sure if the notice should be added to all CMakeLists.txt (it looks like it is not systematically added in the CMakeLists.txt files that are currently in O2).
What is the policy? Should it be added to all files, independently of the extension?

Thanks,
cheers,
Diego

Comment (dberzano) [reply Detectors/MUON/CMakeLists.txt]: At the moment, we only care about `.cxx` and `.h`, as you can [see here](https://github.com/alisw/alidist/blob/master/o2checkcode.sh#L54-L76). So for the moment please take it out.

**Comment by dstocco** [comment]
Hi @dberzano, 
I removed the copyright and pushed. The tests are failing, but the failure is in the TPC simulation...don't think I can do much about it...

Cheers,
Diego

**Comment by dberzano** [comment]
Now they're all OK.

**Pull request merged**